src/componentes/movieCard.tsx

```diff
@@ -0,0 +1,15 @@
+interface Movie {
+  title: string;
+  posterUrl: string;
+}
```

Comment:
Recomendo seguir o padrão do projeto
```suggestion
type MovieCardProps = {
  title: string;
  posterUrl: string;
}
```